### PR TITLE
Add solutions to the CORS error on cloud instances

### PR DESCRIPTION
**Pull request by krzysztofwolski**
Added solution to the issue found on the Cloud instance

### CODE REVIEW

File: docs/developer/extending/apps/developing-apps/troubleshooting.mdx [hunk 1]
```diff
@@ -2,15 +2,30 @@
...
+
+## Client-side GraphQL requests are rejected due to CORS error
+
+If you are using the Saleor Cloud instance, you may encounter the following error when trying to make a GraphQL request from the client side:
+
+```
+Access to fetch at 'https://xxx.eu.saleor.cloud/' from origin 'https://your-application' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
+```
+
+The issue might be caused by the CORS policy set up in the Saleor Cloud dashboard. Navigate to the environment details of your instance and check `Allowed API origins` section. If option `Selected origins` is selected, you have two possible solutions:
```
Comment (2can) [inline docs/developer/extending/apps/developing-apps/troubleshooting.mdx:28]:
```suggestion
The issue might be caused by the CORS policy set up in the Saleor Cloud dashboard. Navigate to the environment details of your instance and check the `Allowed API origins` section. If option `Selected origins` is selected, you have two possible solutions:
```

File: versioned_docs/version-3.x/developer/extending/apps/developing-apps/troubleshooting.mdx [hunk 2]
```diff
@@ -2,17 +2,30 @@
...
+
+## Client-side GraphQL requests are rejected due to CORS error
+
+If you are using the Saleor Cloud instance, you may encounter the following error when trying to make a GraphQL request from the client side:
+
+```
+Access to fetch at 'https://xxx.eu.saleor.cloud/' from origin 'https://your-application' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
+```
+
+The issue might be caused by the CORS policy set up in the Saleor Cloud dashboard. Navigate to the environment details of your instance and check `Allowed API origins` section. If option `Selected origins` is selected, you have two possible solutions:
```
Comment (2can) [inline versioned_docs/version-3.x/developer/extending/apps/developing-apps/troubleshooting.mdx:28]:
```suggestion
The issue might be caused by the CORS policy set up in the Saleor Cloud dashboard. Navigate to the environment details of your instance and check the `Allowed API origins` section. If option `Selected origins` is selected, you have two possible solutions:
```

**Review by 2can — APPROVED** [review]

File: docs/developer/extending/apps/developing-apps/troubleshooting.mdx [hunk 3]
```diff
@@ -2,15 +2,30 @@
...
+
+## Client-side GraphQL requests are rejected due to CORS error
+
+If you are using the Saleor Cloud instance, you may encounter the following error when trying to make a GraphQL request from the client side:
+
+```
+Access to fetch at 'https://xxx.eu.saleor.cloud/' from origin 'https://your-application' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
+```
+
+The issue might be caused by the CORS policy set up in the Saleor Cloud dashboard. Navigate to the environment details of your instance and check the `Allowed API origins` section. If option `Selected origins` is selected, you have two possible solutions:
```
Comment (peelar) [inline docs/developer/extending/apps/developing-apps/troubleshooting.mdx:28]: > Navigate to the environment details of your instance and check the `Allowed API origins` section.

**suggestion**: I think we could use a bit more descriptive instruction here. Can you tell where exactly is this "Allowed API origins" section?

File: docs/developer/extending/apps/developing-apps/troubleshooting.mdx [hunk 4]
```diff
@@ -2,15 +2,30 @@
...
+
+- Add your application URL to the list of allowed origins.
+- Change the implementation of your application to make GraphQL requests from the server side only.
```
Comment (peelar) [inline docs/developer/extending/apps/developing-apps/troubleshooting.mdx:31]: **polish**:

```suggestion
- Ensure your application makes GraphQL requests from the server side only.
```

**Review by peelar — APPROVED** [review]

File: docs/developer/extending/apps/developing-apps/troubleshooting.mdx (hunk 3 above)
Comment (lkostrowski) [reply docs/developer/extending/apps/developing-apps/troubleshooting.mdx:28]: @peelar I think its pretty obvious when you go to cloud console

**Review by lkostrowski — APPROVED** [review]

File: versioned_docs/version-3.x/developer/extending/apps/developing-apps/troubleshooting.mdx [hunk 5]
```diff
@@ -2,17 +2,30 @@
...
+If you are using the Saleor Cloud instance, you may encounter the following error when trying to make a GraphQL request from the client side:
+
+```
+Access to fetch at 'https://xxx.eu.saleor.cloud/' from origin 'https://your-application' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
+```
+
+The issue might be caused by the CORS policy set up in the Saleor Cloud dashboard. Navigate to the environment details of your instance and check the `Allowed API origins` section. If option `Selected origins` is selected, you have two possible solutions:
+
+- Add your application URL to the list of allowed origins.
+- Change the implementation of your application to make GraphQL requests from the server side only.
```
Comment (krzysztofwolski) [inline versioned_docs/version-3.x/developer/extending/apps/developing-apps/troubleshooting.mdx:31]:
```suggestion
- Ensure your application makes GraphQL requests from the server side only.
```

**Pull request merged**